### PR TITLE
[15.0] [OU-ADD] digest

### PR DESCRIPTION
**Pull request by cuongnmtm**
Load noupdate_changes

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

File: openupgrade_scripts/scripts/digest/15.0.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "digest", "15.0.1.1/noupdate_changes.xml")
```
Comment (chienandalu) [inline openupgrade_scripts/scripts/digest/15.0.1.1/post-migration.py:15]: You should remove translations delete translations as well as in https://github.com/OCA/OpenUpgrade/blob/877ae6638a01bcdca7c9281f3e3bb9ca94fd5af5/openupgrade_scripts/scripts/digest/16.0.1.1/post-migration.py#L20

**Review by chienandalu — CHANGES_REQUESTED** [review]
This strikes back in v16 when upgrading mass_mailing, which needs the new hooks provided by these templates

File: openupgrade_scripts/scripts/digest/15.0.1.1/post-migration.py (hunk 1 above)
Comment (cuongnmtm) [reply openupgrade_scripts/scripts/digest/15.0.1.1/post-migration.py:15]: Thanks @chienandalu

**Review by chienandalu — APPROVED** [review]
Thanks :)

**Review by chienandalu — APPROVED** [review]

**Pull request merged**